### PR TITLE
Add OpenRGB Guide

### PR DESCRIPTION
**Pull request by MonsterObserver**

### CODE REVIEW

File: _posts/2023-01-20-install-openrgb.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+sudo mv ~/Downloads/60-openrgb.rules /etc/udev/rules.d/
+```
+
+- After moving the UDEV rules, we need to reload the rules by running the following command:-
+
+```bash
+sudo udevadm control --reload-rules && sudo udevadm trigger
+```
+
+- After reloading the rules, OpenRGB works correctly.
```
Comment (kbdharun) [inline _posts/2023-01-20-install-openrgb.md:45]:
```suggestion
---
title: Install OpenRGB
description: Learn how to install OpenRGB on Vanilla OS.
date: 2023-01-20
layout: article
authors: 
    - MonsterObserver
published: true
---

## Introduction

[**OpenRGB**](https://openrgb.org/) is an essential application for managing your RGB appliances. This guide will help you install the Flatpak version of OpenRGB correctly.

## Prerequisites

- Flatpak: You can find out how to install it [**here**](/2022/12/09/install-flatpaks.html).

## Installing OpenRGB

- You can install OpenRGB from the "Gnome Software" application. Alternatively, you can install it from the command line with the following command:-

```bash
flatpak install flathub org.openrgb.OpenRGB
```

## Setting Up UDEV Rules

For the application to function ideally, add UDEV rules to your system.

- First, install the UDEV rules by clicking [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/jobs/artifacts/master/raw/60-openrgb.rules?job=Linux+64+AppImage&inline=false).

- After downloading the UDEV rules, you need to move them to the correct directory with the following command:-

```bash
sudo mv ~/Downloads/60-openrgb.rules /etc/udev/rules.d/
```

- After moving the UDEV rules, you need to reload the rules by running the following command:-

```bash
sudo udevadm control --reload-rules && sudo udevadm trigger
```

- After reloading the rules, OpenRGB works correctly.
```

File: _posts/2023-01-20-install-openrgb.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+[**OpenRGB**](https://openrgb.org/) is a great application for managing your RGB, however the Flatpak doesn't work correctly out of the box. This guide will help you install it.
+
+## Prerequisites
+
+- Flatpak: You can find out how to install it [**here**](/2022/12/09/install-flatpaks.html).
+
+## Installing OpenRGB
+
+- You can install OpenRGB from the "Gnome Software" application. Alternatively you can install it from the command line with the following command:-
+
```
Comment (kbdharun) [inline _posts/2023-01-20-install-openrgb.md:22]:
```suggestion
---
title: Install OpenRGB
description: Learn how to install OpenRGB on Vanilla OS.
date: 2023-01-20
layout: article
authors: 
    - MonsterObserver
published: true
---

## Introduction

[**OpenRGB**](https://openrgb.org/) is an essential application for managing your RGB appliances. This guide will help you install the Flatpak version of OpenRGB correctly.

## Prerequisites

- Flatpak: You can find out how to install it [**here**](/2022/12/09/install-flatpaks.html).

## Installing OpenRGB

- You can install OpenRGB from the "Gnome Software" application. Alternatively, you can install it from the command line with the following command:-

```

File: _posts/2023-01-20-install-openrgb.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Setting Up UDEV Rules
+
+For the application to function correctly, we need to manually add UDEV rules to our system.
+
+- First, install the UDEV rules by clicking [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/jobs/artifacts/master/raw/60-openrgb.rules?job=Linux+64+AppImage&inline=false).
+
+- After downloading the UDEV rules, we need to move them to the correct directory with the following command:-
+
```
Comment (kbdharun) [inline _posts/2023-01-20-install-openrgb.md:34]:
```suggestion
## Setting Up UDEV Rules

For the application to function ideally, add UDEV rules to your system.

- First, install the UDEV rules by clicking [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/jobs/artifacts/master/raw/60-openrgb.rules?job=Linux+64+AppImage&inline=false).

- After downloading the UDEV rules, you need to move them to the correct directory with the following command:-

```

File: _posts/2023-01-20-install-openrgb.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+```bash
+sudo mv ~/Downloads/60-openrgb.rules /etc/udev/rules.d/
+```
+
+- After moving the UDEV rules, we need to reload the rules by running the following command:-
```
Comment (kbdharun) [inline _posts/2023-01-20-install-openrgb.md:39]:
```suggestion
- After moving the UDEV rules, you need to reload the rules by running the following command:-
```

**Review by kbdharun — COMMENTED** [review]
Minor suggestions, feel free to update it.

File: _posts/2023-01-20-install-openrgb.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+- You can install OpenRGB from the "Gnome Software" application. Alternatively, you can install it from the command line with the following command:-
+
+```bash
+flatpak install flathub org.openrgb.OpenRGB
+```
+
+## Setting Up UDEV Rules
+
+For the application to function ideally, add UDEV rules to your system.
```
Comment (MonsterObserver) [inline _posts/2023-01-20-install-openrgb.md:29]:
```suggestion
For the application to function, add UDEV rules to your system.
```

File: _posts/2023-01-24-install-openrgb.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+sudo abroot exec apt install libfuse2
+```
+
+- After the installation is complete reboot the system. You can do this by typing `reboot`.
+
+### Installing OpenRGB
+
+- Install "Linux 64-bit" from the "Linux Binaries (AppImage)" section [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/releases/permalink/latest#Linux-64-bit).
+
+- Go to the directory where the AppImage is installed and make it an executable with these commmands:-
```
Comment (kbdharun) [inline _posts/2023-01-24-install-openrgb.md:45]:
```suggestion
- Go to the directory where the AppImage is installed and make it an executable with these commands:-
```

File: _posts/2023-01-24-install-openrgb.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+
+### Prerequisites
+
+- AppImage Support: If you enabled AppImage in the "First Setup" when installing Vanilla OS, you won't have to do anything. If you didn't enable AppImage, run this command:-
+
+```bash
+sudo abroot exec apt install libfuse2
+```
+
+- After the installation is complete reboot the system. You can do this by typing `reboot`.
```
Comment (kbdharun) [inline _posts/2023-01-24-install-openrgb.md:39]:
```suggestion
- After the successful installation, reboot your system. You can do this by typing `reboot`.
```

File: _posts/2023-01-24-install-openrgb.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+```bash
+flatpak install flathub org.openrgb.OpenRGB
+```
+
+## AppImage
+
+### Prerequisites
+
+- AppImage Support: If you enabled AppImage in the "First Setup" when installing Vanilla OS, you won't have to do anything. If you didn't enable AppImage, run this command:-
```
Comment (kbdharun) [inline _posts/2023-01-24-install-openrgb.md:33]:
```suggestion
- AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. Else, run this command:-
```

**Review by kbdharun — APPROVED** [review]
LGTM after this

File: _posts/2023-01-24-install-openrgb.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+
+```bash
+flatpak install flathub org.openrgb.OpenRGB
+```
+
+## AppImage
+
+### Prerequisites
+
+- AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. Or else, run this command:-
```
Comment (MonsterObserver) [inline _posts/2023-01-24-install-openrgb.md:33]:
```suggestion
- AppImage Support: If you have enabled AppImage in the "First Setup" when installing Vanilla OS, this step isn't necessary. If you didn't enable it, run the following command:-
```

File: _posts/2023-01-24-install-openrgb.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+sudo abroot exec apt install libfuse2
+```
+
+- After the successful installation, reboot your system. You can do this by typing `reboot`.
+
+### Installing OpenRGB
+
+- Install "Linux 64-bit" from the "Linux Binaries (AppImage)" section [**here**](https://gitlab.com/CalcProgrammer1/OpenRGB/-/releases/permalink/latest#Linux-64-bit).
+
+- Go to the directory where the AppImage is installed and make it an executable with these commands:-
```
Comment (MonsterObserver) [inline _posts/2023-01-24-install-openrgb.md:45]:
```suggestion
- Go to the directory where the AppImage is installed and make it an executable with the following commands:-
```

**Comment by mirkobrombin** [comment]
is this ready?

**Comment by MonsterObserver** [comment]
> is this ready?

Yeah it's GTG

**Pull request merged**